api/client/lib.py

```diff
@@ -349,12 +394,77 @@ def rank_series_by_source(access_token, api_host, series_list):
             yield series_with_source
 
 
+def format_list_of_series(series_list):
```

Comment:
function name sounds like it goes the other way.. how about list_of_series_to_single_series

Reply:
Done